erpnext/payroll/doctype/overtime_slip/overtime_slip.json

```
@@ -0,0 +1,196 @@
+{
+ "actions": [],
+ "autoname": "HR-OVR-SLIP-.#####",
```

```suggestion
 "autoname": "HR-OT-SLIP-.#####",
```
Better naming series since OT is short for Overtime?

---

Done